tests/validators/test_frozenset.py

```diff
@@ -71,6 +71,13 @@ def test_frozenset_no_validators_both(py_and_json: PyAndJson, input_value, expec
         ((1, 2, 3, 2, 3), frozenset({1, 2, 3})),
         ((), frozenset()),
         (frozenset([1, 2, 3, 2, 3]), frozenset({1, 2, 3})),
+        pytest.param(
```

Comment:
Best to remove this since it it doesn't make sense.

Instead add two new tests:
* A test with field `List[Any]` showing a list of tuples is the output for `{1: 10, 2: 20, 3: 30}.items()`
* A test with field `List[Tuple[int, int]]` or similar showing `{1: 10, 2: 20, '3': '30'}.items()` works.

Reply:
same for set, frozenset and tuple.

Reply:
Thanks for the pointers. I'll follow up if I have any questions!